nodes/test/approval/test_approval.py

```
@@ -0,0 +1,719 @@
+
+# ---------------------------------------------------------------------------
+# Bootstrap: stub out rocketlib and ai.common.* so the node can be imported
+# without the full engine.
+# ---------------------------------------------------------------------------
+
+NODES_SRC = Path(__file__).parent.parent.parent / 'src' / 'nodes'
+if str(NODES_SRC) not in sys.path:
+    sys.path.insert(0, str(NODES_SRC))
+
+
+def _stub_rocketlib():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_stub_rocketlib`

(ANN202)